packages/cfd/src/Containers/cfd-password-modal/cfd-password-form.tsx

```diff
@@ -0,0 +1,264 @@
+        setFieldTouched('password', true);
+    });
+};
+
+export const CFDPasswordForm = ({
+    account_title,
+    account_type,
+    closeModal,
+    error_message,
+    error_type,
+    form_error,
+    has_mt5_account,
+    is_real_financial_stp,
+    jurisdiction_selected_shortcode,
+    onCancel,
+    onForgotPassword,
+    platform,
+    should_set_trading_password,
+    show_eu_related_content,
+    submitPassword,
+    validatePassword,
+}: TCFDPasswordFormProps) => {
+    const button_label = React.useMemo(() => {
+        if (error_type === 'PasswordReset') {
+            return localize('Try later');
+        }
+        return localize('Add account');
+    }, [error_type]);
+
+    const has_cancel_button = (isDesktop() ? !should_set_trading_password : true) || error_type === 'PasswordReset';
+
+    const cancel_button_label = getCancelButtonLabel({ should_set_trading_password, error_type });
+
+    const handleCancel = () => {
+        if (!has_cancel_button) {
+            return undefined;
+        }
+        if (should_set_trading_password) {
+            return onCancel();
+        }
+
+        return onForgotPassword();
+    };
+
+    if (error_type === 'PasswordReset') {
+        return (
+            <React.Fragment>
+                <div className='cfd-password-reset'>
+                    <div className='cfd-password-modal__content cfd-password-modal__content--password-reset'>
+                        <Text as='p' line_height='24' size='xs'>
+                            <Localize i18n_default_text='Please try again in a minute.' />
+                        </Text>
+                    </div>
+                    <Formik onSubmit={closeModal} initialValues={{}}>
+                        {({ handleSubmit }) => (
+                            <form onSubmit={handleSubmit}>
+                                <FormSubmitButton
+                                    has_cancel={has_cancel_button}
+                                    cancel_label={cancel_button_label}
+                                    onCancel={handleCancel}
+                                    is_absolute={isMobile()}
+                                    label={button_label}
+                                />
+                            </form>
+                        )}
+                    </Formik>
+                </div>
+            </React.Fragment>
+        );
+    }
+
+    if (should_set_trading_password) {
+        return (
+            <CFDCreatePasswordForm
+                platform={platform}
+                error_message={error_message}
+                validatePassword={validatePassword}
+                submitPassword={submitPassword}
+                has_mt5_account={has_mt5_account}
+                is_real_financial_stp={is_real_financial_stp}
+            />
+        );
+    }
+
+    const accountTitle = () => {
+        switch (platform) {
+            case 'ctrader':
+            case 'derivez':
+            case 'dxtrade':
```

Comment:
```suggestion
            case CFD_PLATFORMS.CTRADER:
            case CFD_PLATFORMS.DXTRADE:
```
We can remove this Derivez as this is going to be removed
And use the enums object instead of hardcoded
strings